content/en/news/releases/0.x/announcing-0.1/index.md

```diff
@@ -16,7 +16,7 @@ aliases:
 ---
 
 Google, IBM, and Lyft are proud to announce the first public release of [Istio](/): an open source project that provides a uniform way to connect, secure, manage and monitor microservices. Our current release is targeted at the [Kubernetes](https://kubernetes.io/) environment; we intend to add support for other environments such as virtual machines and Cloud Foundry in the coming months.
-Istio adds traffic management to microservices and creates a basis for value-add capabilities like security, monitoring, routing, connectivity management and policy.  The software is built using the battle-tested [Envoy](https://envoyproxy.github.io/envoy/) proxy from Lyft, and gives visibility and control over traffic *without requiring any changes to application code*. Istio gives CIOs a powerful tool to enforce security, policy and compliance requirements across the enterprise.
+Istio adds traffic management to microservices and creates a basis for value-add capabilities like security, monitoring, routing, connectivity management and policy.  The software is built using the battle-tested [Envoy](https://www.envoyproxy.io/docs/envoy/) proxy from Lyft, and gives visibility and control over traffic *without requiring any changes to application code*. Istio gives CIOs a powerful tool to enforce security, policy and compliance requirements across the enterprise.
```

Comment:
i think you need to just link to https://www.envoyproxy.io/docs - the extra part of the url is currently redundant - perhaps we should add a page there

Reply:
maybe  https://www.envoyproxy.io/docs/latest

Reply:
hmm, latest seems to be pre-release, so not the best

Reply:
They seem to have changed their URL structure as well, and there are ~ 200 links out to that domain. Would you prefer that we do a naive find and replace so it's linking to the right domain, or does someone need to spend a few hours checking every link?

Reply:
e.g. I can't find a page for <https://www.envoyproxy.io/docs/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto>

Reply:
try https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto

Reply:
Links don't work for me either.

Reply:
https://www.envoyproxy.io/docs/envoy/v1.27.0/ works, but we are likely forcing a particular version. Maybe we can add a variable to the data/arg file for the released Envoy version (would update in sync with Istio releases) and add that to the link.

Reply:
> e.g. I can't find a page for https://www.envoyproxy.io/docs/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto

https://www.envoyproxy.io/docs/envoy/v1.27.0/api-v3/extensions/transport_sockets/tls/v3/common.proto

Reply:
> They seem to have changed their URL structure as well, and there are ~ 200 links out to that domain. Would you prefer that we do a naive find and replace so it's linking to the right domain, or does someone need to spend a few hours checking every link?

I think you can enable link checking during a local lint to verify links. You'll find a number that don't work which is preventing the link checker from being re-enabled in the pipeline. But should be Abel to at lease verify those related to envoy.

Reply:
@ericvn mind pointing me to the docs for running that linter and adding that config option? Neither the contributing guide nor the readme mention it
Thanks!

Reply:
Looking at scripts/lint_site.sh and the Makefile, I think `CHECK_EXTERNAL_LINKS=true make lint-en` would run the check against external links.

Reply:
In the output for the above I see:
```
[404] https://envoyproxy.github.io/envoy/
```
and many more.

Reply:
iiuc these updates are for historical blog posts/release announcements

in which case surely rather than `latest` or `1.27.x` they should point to what is contemporaneous with the posts 

all of our historical docs are there and should be exactly as they were at the time

(fwiw i dont think the site structure changed that much since these posts were made - but im not sure it matters)

going forward i would strongly suggest that any links point to the current (at the time) version of the docs

Reply:
They weren't trying to point to the docs, they were trying to point to the Envoy project.  I've updated them all to use the `{{glossary}}` shortcode throughout.